packages/web/components/Quest/QuestForm.tsx

```diff
@@ -164,11 +167,25 @@ export const QuestForm: React.FC<Props> = ({
   });
   const router = useRouter();
   const [exitAlert, setExitAlert] = useState<boolean>(false);
+  const [previewImg, setPreviewImage] = useState<string>('');
   const createQuestInput = watch();
 
+  function showImagePreview(e: ChangeEvent<HTMLInputElement>) {
+    if (!e.target || !e.target.files || !e.target.files[0]) {
```

Comment:
```suggestion
    if (!e.target?.files?.[0]) {
```

Reply:
Yeaaah, I tried this in my OG code and it kept throwing errors 🤷🏽‍♂️so thats why I wrote it the verbose way lol

Reply:
Maybe it was before you had the typing right?

I made `const file = e.target?.files?.[0]` the first line, then check `if(!file)` and TypeScript is happy with it.